proto/import_sstpb.proto

```diff
@@ -394,6 +395,9 @@ message KVMeta {
 
     // the compression type for the file.
     backup.CompressionType compression_type = 13;
+
+    // encryption information of the kv file, not set if encryption is not enabled.
+    encryptionpb.FileEncryptionInfo file_encryption_info = 19;
```

Comment:
The largest tag number in this message is 13. Why 19?

Reply:
good catch!